apps/src/lib/config/mod.rs

```diff
@@ -83,7 +83,7 @@ pub struct Ledger {
     pub chain_id: ChainId,
     pub shell: Shell,
     pub tendermint: Tendermint,
-    pub ethereum: ethereum::Config,
+    pub ethereum_bridge: ethereum_bridge::ledger::Config,
```

Comment:
Oh wait, you do have this as a config. We can't configure minimum confirmations or addresses in a normal config. They have to be set at genesis and changed only with governance votes.

Reply:
ah okay that makes sense, since you can run into consensus failures if two validators have different ideas of what the min number of blocks should be for an event to be considered as valid.

Reply:
With the Tendermint v0.37 design, we shouldn't get consensus failures if validators have different ideas of min confirmations. For the addresses of the contracts, those should change only via hard fork as far as I understand it, if we do a contract upgrade, though technically it would be possible for a a governance proposal to change them (right now as the code currently is).

Reply:
Even if it didn't lead to consensus failures, it is supposed to be that all validators agree on this value. Otherwise, events may get skipped, or someone trying to slash some one else, or generally undesired behavior. Anything that is protocol specified shouldn't be in a config file for running the node software, it should only be in a genesis config.